staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_aggregator.go

```diff
@@ -0,0 +1,253 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apiserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"sort"
+
+	"github.com/emicklei/go-restful"
+	"github.com/go-openapi/spec"
+	"k8s.io/apiserver/pkg/authentication/user"
```

Comment:
I fear it's a bit mixed. Most code probably has "stdlib, 3rdparty, kube". Some of the staging repos (mostly by @deads2k I think) have "stdlib, 3rdparty, kube, that very staging repo". Both go from near-to-standard-go to concrete. Your order was mostly that of david, but the apiserver package in both blocks was misplaced. Anyway, it's only cosmetics :)